tests/unit/main_html5/wrapper-api-tests.js

```diff
@@ -118,7 +119,10 @@ describe('main_html5 wrapper tests', function () {
 
   it('should clear closed captions when setting a new url', function(){
     wrapper.setVideoUrl("url");
+    $(element).triggerHandler("loadedmetadata");
+    $(element).triggerHandler("canplay");
     wrapper.setClosedCaptions(language, closedCaptions, params);
+    element.textTracks.onaddtrack();
```

Comment:
Do we have a listener for onaddtrack in our video plugin? I'm just curious how this is used for the unit tests.